src/vscode-bicep/src/commands/build.ts

```diff
@@ -14,25 +15,14 @@ export class BuildCommand implements Command {
   ) {}
 
   public async execute(
-    _context: IActionContext,
+    context: IActionContext,
     documentUri?: vscode.Uri | undefined
   ): Promise<void> {
-    documentUri ??= vscode.window.activeTextEditor?.document.uri;
-
-    if (!documentUri) {
-      return;
-    }
-
-    if (documentUri.scheme === "output") {
```

Comment:
We need to put this check in findOrCreateActiveBicepFile(..). This is a valid issue.

Reply:
It's no longer relevant because when we use the active editor, we always check for langid = bicep:

```

  if (options?.considerActiveEditor) {
    const activeEditor = window.activeTextEditor;
    if (activeEditor?.document?.languageId === "bicep") {
      properties.targetFile = "activeEditor";
      return activeEditor.document.uri;
    }
  }
```

If langid is bicep, scheme can't be output.

Reply:
I am hitting this issue when I invoke build command with focus on output window:
![image](https://user-images.githubusercontent.com/30270536/160474199-eb15fa12-7c46-46c6-a3a1-6d3643f1b0ee.png)

I think we were handling this scenario before.

Reply:
How are you invoking it?  I don't see this.

Reply:
And what workspace/folder/etc do you have open?

Reply:
Repro steps:
- Open bicep output channel
- Right click on output channel
- Invoke build command

Let me know if you don't see the error. Thanks!

Reply:
Issue moved to https://github.com/Azure/bicep/issues/6338